components/board/status.go

```diff
@@ -5,42 +5,42 @@ import (
 	"fmt"
 
 	"github.com/pkg/errors"
-	commonpb "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/board/v1"
 )
 
 // CreateStatus constructs a new up to date status from the given board.
 // The operation can take time and be expensive, so it can be cancelled by the
 // given context.
-func CreateStatus(ctx context.Context, b Board, extra map[string]interface{}) (*commonpb.BoardStatus, error) {
-	var status commonpb.BoardStatus
+func CreateStatus(ctx context.Context, b Board) (*pb.Status, error) {
+	var status pb.Status
 
 	if names := b.AnalogNames(); len(names) != 0 {
```

Comment:
[note to self] status is using the board interface to get the analog/interrupt names, while the client code tracks pin names from a requests, since the name list isn't a part of the proto/api.